api/messages/struct.proto

```diff
@@ -59,18 +59,28 @@ message Value {
   oneof kind {
     // Represents a null value.
     NullValue null_value = 1;
-    // Represents a double value.
-    double number_value = 2;
+    // Represents a float64 value.
+    double float64_value = 2;
+    // Represents a float32 value
+    float float32_value = 3;
+    // Represents an int32 value
+    int32 int32_value = 4;
+    // Represents an int64 value
+    int64 int64_value = 5;
+    // Represents a uint32 value
+    uint32 uint32_value = 6;
+    // Represents a uint64 value
+    uint64 uint64_value = 7;
     // Represents a string value.
```

Comment:
The idiomatic way to do these kind of changes is to comment the top of the proto with the next unused id, always select from that increasing value when adding fields, and never change the id of old fields or reuse them if deleted. That limits how catastrophic the consequences are if two binaries built against different versions try to talk to each other (there may be unrecognized fields but at least it won't try deserializing raw bytes into the wrong type). We're still prerelease so we have some room to make gratuitous incompatible changes but it's probably still a good habit, and doesn't affect any human-written code.

Reply:
Yah, my hope was that we're early enough in prerelease that I could get away with not doing that, and keep the values a little "cleaner", for whatever it's worth. Also wasn't sure if it mattered at all, since I needed to remove the original `number` value.